plugins/admincmd.sma

```diff
@@ -165,18 +164,18 @@ public plugin_init()
 	register_clcmd("amx_showrcon", "cmdShowRcon", ADMIN_RCON, "<command line>")
 	register_clcmd("pauseAck", "cmdLBack")
 
-	rcon_password=get_cvar_pointer("rcon_password");
-	pausable=get_cvar_pointer("pausable");
-	timelimit=get_cvar_pointer( "mp_timelimit" );
-	p_amx_tempban_maxtime = register_cvar("amx_tempban_maxtime", "4320", FCVAR_PROTECTED);
+	g_pRconPassword=get_cvar_pointer("g_pRconPassword");
```

Comment:
Are you sure you want to replace ("rcon_password") and other cvar names? :P

Reply:
You're viewing an older commit, that was a mistake I made which was fixed later.

Reply:
Oh, my mistake ;)

Reply:
Actually I had it mistaken in another place in the code and I fixed it now, thanks for reminding me. The search and replace function failed me once again.